edb/lib/std/30-strfuncs.eql

```diff
@@ -91,3 +91,86 @@ std::`>` (l: std::str, r: std::str) -> std::bool
 CREATE INFIX OPERATOR
 std::`>=` (l: std::str, r: std::str) -> std::bool
     FROM SQL OPERATOR r'>=';
+
+
+CREATE FUNCTION
+std::ord(val: std::str) -> std::int64
```

Comment:
I'd drop `ord()` and `chr()`.  I only used them a few times in my life and I fail to come up with a good example of how they can be used in EdgeQL.

Reply:
Dropped.